engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/runtime/ReleaseReferencesNode.java

```diff
@@ -0,0 +1,35 @@
+package org.enso.interpreter.node.expression.builtin.runtime;
+
+import com.oracle.truffle.api.CompilerDirectives;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.EnsoContext;
+
+@BuiltinMethod(
+    type = "Runtime",
+    name = "release_references",
```

Comment:
Oh, oh, a new builtin!

I tried to keep the _conceptual surface_ of the API as _small as possible_ and exposing `release_references` to general use goes against that.

I can see why it is useful in unit tests, but I hope we'll find another solution to the problem than a _builtin available to everyone_.